### DISCUSSION TITLE
Bagaimana cara integerasikan Codeigniter 3 dengan RabbitMQ ?

### DISCUSSION THREAD

**Discussion by rmdhfz** [community/community#55263] [Discussions] [answered]
### Select Topic Area

Question

### Body

Halo semua,
Saya ingin berdiskusi sekaligus bertanya bagaimana cara integerasi codeigniter 3 dengan rabbitmq, sekaligus ini adalah pertanyaan pertama saya di GitHub Discussions. Terima kasih!

**Comment by refs05** [comment]
ya silahkan mas hafiz

**Comment by refs05** [comment]
Silahkan buka chat GPT

**Comment by refs05** [comment, marked as answer]
mungkin bisa menggunakan ini https://github.com/romainrg/rabbitmq_client

**Comment by rmdhfz** [comment]
Terima kasih mas @refs05
